src/test/resources/config_in_admin_format.json

```diff
@@ -64,11 +64,21 @@
 				"accessibleByPerRequestKey": true,
 				"contentPartsSupported": false,
 				"temperatureSupported": true,
-				"addonsSupported": false
+				"addonsSupported": false,
+                "cacheSupported": false,
```

Comment:
indent are differ with other lines in the file

Reply:
Looks like GitHub UI issue. Try to download the file and view it - indentation is correct